Makefile

```diff
@@ -164,3 +164,7 @@ install-konduit: ## Install the konduit script, for accessing backend services
 		&& curl -s https://raw.githubusercontent.com/DFE-Digital/teacher-services-cloud/master/scripts/konduit.sh -o bin/konduit.sh \
 		&& chmod +x bin/konduit.sh \
 		|| true
+
+.PHONY: terraform-refresh
```

Comment:
out of curiosity why do we need this step?

Reply:
Not entirely sure, I think we added it while on the call with Colin. Will make a note to check it and we'll remove if unnecessary/unuseful.

Perhaps it was related to the extra steps caused by needing to request PIM approval then log out/in?